PB-IDE-Tools/displayASMcode.pb

```diff
@@ -0,0 +1,308 @@
+﻿;-Description   : Displays the commented ASM code from the PB compiler
+; OS            : Windows, Linux, Mac
+; English-Forum :
+; French-Forum  :
+; German-Forum  : https://www.purebasic.fr/german/viewtopic.php?f=10&t=30935
+; 
+; Author        : Sicro
+; Author second : mk-soft
+; Update        : 28.11.2021
```

Comment:
I don't like the author, author second and update lines and the minus character before word _Description_.